pkg/directory/directory_test.go

```diff
@@ -1,49 +1,54 @@
 package directory
 
 import (
+	"io/ioutil"
 	"os"
 	"testing"
 )
 
 // Size of an empty directory should be 0
 func TestSizeEmpty(t *testing.T) {
+	var dir string
 	var err error
-	if err = os.Mkdir("/tmp/testSizeEmptyDirectory", 0777); err != nil {
+	if dir, err = ioutil.TempDir(os.TempDir(), "testSizeEmptyDirectory"); err != nil {
```

Comment:
That's really a detail, but you can give `TempDir` an empty string as first argument and let him figure out the default.